### ISSUE TITLE
Error in build (postcss_1.default is not a function)

### ISSUE THREAD

**Issue by bayssmekanique** [vuejs/vuepress#268]
This error appeared after I cleared my modules directory and reinstalled from scratch (no `package-lock.json`).  I cleared npm cache, and tested on a completely blank project with only a single `readme.md` file with no configuration changes and it fails in the exact same way.

OS: Window 10
Node: 9.5.0
VuePress: 0.8.4
Local install (npm)

Error message:
```
ERROR in ./site/.vuepress/theme/Page.vue?vue&type=style&index=0&lang=stylus (./node_modules/css-loader??ref--11-oneOf-1-1!./node_modules/vue-loader/lib/loaders/stylePostLoader.js!./node_modules/postcss-loader/lib??ref--11-oneOf-1-2!./node_modules/stylus-loader??ref--15-oneOf-1-3!./node_modules/vue-loader/lib??vue-loader-options!./site/.vuepress/theme/Page.vue?vue&type=style&index=0&lang=stylus)
Module build failed: TypeError: postcss_1.default is not a function
    at compileStyle (D:\Development\Keiser\Keiser.Web\Keiser.Web.Dev\node_modules\@vue\component-compiler-utils\dist\compileStyle.js:34:35)
    at Object.module.exports (D:\Development\Keiser\Keiser.Web\Keiser.Web.Dev\node_modules\vue-loader\lib\loaders\stylePostLoader.js:9:33)
```

**Comment by bayssmekanique** [comment]
I just tested on a second computer (also Windows 10, Node 9.5.0) and it shows the same error with a new project.

**Comment by ulivz** [comment]
Have you tried with `yarn`?

**Comment by bayssmekanique** [comment]
I just tried with `yarn`  version `1.6.0` and have the same error.

**Comment by dolare** [comment]
same error

**Comment by ulivz** [comment]
Maybe I need to borrow a windows to repro that. 😅

**Comment by mattixittam** [comment]
Tried it with several versions of VuePress, clean install each time. Same error.

```
Module build failed: TypeError: postcss_1.default is not a function
    at compileStyle (/mnt/c/CODE/vuepress/node_modules/@vue/component-compiler-utils/dist/compileStyle.js:34:35)
    at Object.module.exports (/mnt/c/CODE/vuepress/node_modules/vue-loader/lib/loaders/stylePostLoader.js:9:33)
```

OS: Windows 10 (Insider build 17134), using Ubuntu 16.04 via WSL
Node: 8.11.1
VuePress: 0.4.1, 0.6.1, 0.7.0, 0.8.4
Local install (yarn)

**Comment by bayssmekanique** [comment]
I just tested on a Linux box and have the same error.

Ubuntu 16.04.03
Node 10.0.0

**Comment by bayssmekanique** [comment]
This issue didn't exist yesterday when I performed a clean install, so it's most likely the result of a release from the last day or so.

**Comment by bayssmekanique** [comment]
I'm fairly certain the bug is in `vuejs/component-compiler-utils` @ https://github.com/vuejs/component-compiler-utils/commit/1456e3dc07c260dad276356be778a0dcb9fc6787

**Comment by dolare** [comment]
But it seems like the previous version also can not work?

**Comment by bayssmekanique** [comment]
@dolare, the previous version of what is not working?  `vuepress` or `component-compilter-utils`?

I have been able to get it working by modifying line 34 of the `compileStyle.js` file and removing the `.default` making the line `result = postcss_1(plugins).process(source, postCSSOptions);`.  I'm guessing the issue is somewhere in the build process for that library.

**Comment by ulivz** [comment]
Reported issue has been sent to **@component-compilter-utils**. Please wait patiently.

**Comment by dolare** [comment]
@bayssmekanique Thanks. I temporarily solved this issue like what you did.

**Comment by filipekiss** [comment]
@dolare @bayssmekanique 

Instead of messing up with `compileStyle.js`, add this to your `package.json`:

```json
    "resolutions": {
        "@vue/component-compiler-utils": "1.1.0"
    }
```

This will use the previous version of `@vue/component-compiler-utils` and everything will work as expected!

![image](https://user-images.githubusercontent.com/48519/39333770-77bbed88-4982-11e8-9095-cdc1aa663770.png)

**Comment by yyx990803** [comment]
Fixed in `@vue/component-compiler-utils@1.2.1`

**Comment by dolare** [comment]
@filipekiss Thanks for letting me know that.
